addons/mail/static/src/components/attachment_list/attachment_list.xml

```diff
@@ -6,12 +6,12 @@
             <t t-if="attachmentList">
                 <div class="o_AttachmentList_partialList o_AttachmentList_partialListImages">
                     <t t-foreach="attachmentList.attachmentImages" t-as="attachmentImage" t-key="attachmentImage.localId">
-                        <AttachmentImage class="o_AttachmentList_attachment" attachmentImageLocalId="attachmentImage.localId"/>
+                        <AttachmentImage class="o_AttachmentList_attachment" attachmentImageLocalId="attachmentImage.localId" onAttachmentRemoved="props.onAttachmentRemoved"/>
```

Comment:
There is way too much noise in all components for these, we should make sure to use fields instead when the view model already exist.

Reply:
It's delicate to turn into fields.
The event bubbling were business logic with structure of components.
Passing props keeps business logic with structure of components, so it's easy and safe.

I'm not confident in properly converting them to models & fields because it's challenging on a case-by-case, some of which could be very hard and require lots of time / more diff.